DisplaySettings/doc/DisplaySettingsPlugin.md

```diff
@@ -3408,58 +3408,8 @@ Possible values:
 }
 ```
 
-<a name="method.setSoundMode"></a>
```

Comment:
Was this intended to be removed? There is no setSoundMode description now with the new changes.

Reply:
good catch. This commit was not intended to be part of this pull request. I'm closing the pull request and have opened #2029 which removes the stale commit.